smithy-aws-apigateway-traits/build.gradle.kts

```diff
@@ -16,10 +16,10 @@
 description = "This module provides Smithy traits and validators for API Gateway."
 
 ext {
-    displayName = "Smithy :: AWS :: API Gateway Traits"
-    moduleName = "software.amazon.smithy.aws.apigateway.traits"
+    set("displayName", "Smithy :: AWS :: API Gateway Traits")
```

Comment:
I think these should likely be `extra` properties rather than extensions. Interested if you found any guidance in the gradle guides that may suggest otherwise.

Reply:
I see, I found an interesting explanation here: 
https://stackoverflow.com/a/56306736

Seeing this it makes sense for us to use `extra`. 

Previously I used :

```kotlin
extra.apply{
   set("displayName", " some display name")
   set("moduleName", "some module name")
}

```

and it generated some weird directory named `property(fixed` in all subprojects.


I will use :

```kotlin
extra["displayName"] = "Smithy :: Waiters"
extra["moduleName"] = "software.amazon.smithy.waiters"

```